app/utils/callback/lava.py

```diff
@@ -457,6 +458,33 @@ def _add_test_results(group, suite_results, suite_name):
     })
 
 
+def _add_test_info(group):
+    """Add rootfs info
+
+    Download the the json file with the information of the rootfs if it's
+    available and add its information to the group data
+
+    :param group: Test group data.
+    :type group: dict
+    """
+
+    rootfs_url = group["initrd"]
```

Comment:
Note: This would fail in the case where a job ran without any `initrd` such as the current `boot-nfs` test plan.  However, we're lucky as the `boot-nfs` test plan doesn't actually have any test cases so this function isn't called.

For the time being, I think it would be safer to do this:
```
rootfs_url = group.get('initrd')
if not rootfs_url:
    return
```

Then later on, add support for other rootfs types i.e. nfs...

Reply:
Updated and tested locally. Thanks!